p2p/conn/connection.go

```diff
@@ -524,14 +518,16 @@ func (c *MConnection) Status() ConnectionStatus {
 	status.SendMonitor = c.sendMonitor.Status()
 	status.RecvMonitor = c.recvMonitor.Status()
 	status.Channels = make([]ChannelStatus, len(c.channels))
-	for i, channel := range c.channels {
+	i := 0
+	for _, channel := range c.channels {
```

Comment:
it does mean the channels will be in a different order here every time over the RPC. Could be weird.

Reply:
I would change RPC to return a map instead. I.e. if somebody relies on order here, then he/she is obviously doing wrong thing

Reply:
Like https://github.com/tendermint/tendermint/pull/1141/commits/da59c46a5a0589803aaaebdcb4624788cba430ff